Apps/AdminWebClient/test/unit/Services.Test/CovidSupportServiceTests.cs

```diff
@@ -0,0 +1,211 @@
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.AdminWebClientTests.Services.Test
+{
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Net.Http;
+    using HealthGateway.Admin.Server.Api;
+    using HealthGateway.Admin.Server.Delegates;
+    using HealthGateway.Admin.Server.Models.CovidSupport;
+    using HealthGateway.Admin.Services;
+    using HealthGateway.Common.AccessManagement.Authentication;
+    using HealthGateway.Common.Data.Constants;
+    using HealthGateway.Common.Data.Models.ErrorHandling;
+    using HealthGateway.Common.Data.ViewModels;
+    using HealthGateway.Common.Delegates;
+    using HealthGateway.Common.Delegates.PHSA;
+    using HealthGateway.Common.Services;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+    using Moq;
+    using Refit;
+    using Xunit;
+
+    /// <summary>
+    /// Unit Tests for the Lab Test Kit registration service.
+    /// </summary>
+    public class CovidSupportServiceTests
+    {
+        private const string AccessToken = "access_token";
+        private const string Phn = "9735353315";
+
+        /// <summary>
+        /// Tests a various http status codes on Covid Assessment Details.
+        /// </summary>
+        /// <param name="httpStatusCode">The http status code to return from the mock.</param>
+        /// <param name="resultStatus">The result code to return from the mock.</param>
+        [Theory]
+        [InlineData(HttpStatusCode.OK, ResultType.Success)]
+        [InlineData(HttpStatusCode.NoContent, ResultType.Error)]
+        [InlineData(HttpStatusCode.Unauthorized, ResultType.Error)]
+        [InlineData(HttpStatusCode.Forbidden, ResultType.Error)]
+        [InlineData(HttpStatusCode.MethodNotAllowed, ResultType.Error)]
+        public void GetCovidAssessmentDetails(HttpStatusCode httpStatusCode, ResultType resultStatus)
+        {
+            CovidAssessmentDetailsResponse expectedResult = new();
+            RequestResult<CovidAssessmentDetailsResponse> actualResult = GetCovidSupportService(expectedResult, httpStatusCode, false).GetCovidAssessmentDetailsAsync(Phn).Result;
+            Assert.True(actualResult.ResultStatus == resultStatus);
```

Comment:
Nit: I think Assert.Equal() and similar methods are preferable to Assert.True() because they provide more contextual error messages when they fail.

Reply:
I'll keep that in mind for future tests.